[2019-11-17 09:12:17,350] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_dag.Load_dependencies_dim_table 2018-11-03T16:00:00+00:00 [queued]>
[2019-11-17 09:12:17,356] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_dag.Load_dependencies_dim_table 2018-11-03T16:00:00+00:00 [queued]>
[2019-11-17 09:12:17,356] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 09:12:17,356] {taskinstance.py:839} INFO - Starting attempt 3 of 4
[2019-11-17 09:12:17,356] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 09:12:17,361] {taskinstance.py:859} INFO - Executing <Task(LoadDimensionOperator): Load_dependencies_dim_table> on 2018-11-03T16:00:00+00:00
[2019-11-17 09:12:17,361] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'capstone_dag', 'Load_dependencies_dim_table', '2018-11-03T16:00:00+00:00', '--job_id', '206', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/capstone_dag.py', '--cfg_path', '/var/folders/8q/nyxhvd6d55nfqjdbrybk6jpw0000gn/T/tmpu7igdf2d']
[2019-11-17 09:12:18,085] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table /Users/likejun/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2019-11-17 09:12:18,086] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   """)
[2019-11-17 09:12:18,164] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table [2019-11-17 09:12:18,164] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-11-17 09:12:18,464] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table [2019-11-17 09:12:18,464] {dagbag.py:90} INFO - Filling up the DagBag from /Users/likejun/airflow/dags/capstone_dag.py
[2019-11-17 09:12:18,477] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table [2019-11-17 09:12:18,477] {cli.py:516} INFO - Running <TaskInstance: capstone_dag.Load_dependencies_dim_table 2018-11-03T16:00:00+00:00 [running]> on host likejundeMacBook-Pro.local
[2019-11-17 09:12:18,483] {load_dimensions.py:23} INFO - LoadDimensionOperator for dependencies
[2019-11-17 09:12:18,488] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:12:18,488[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-11-17 09:12:21,031] {load_dimensions.py:34} INFO - get 0 records from dependencies
[2019-11-17 09:12:21,034] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:12:21,034[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-11-17 09:12:24,308] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:12:24,308[0m] {[34mdbapi_hook.py:[0m171} INFO[0m - DELETE FROM dependencies[0m
[2019-11-17 09:12:24,831] {taskinstance.py:1051} ERROR - relation "dependencies" does not exist
Traceback (most recent call last):
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 38, in execute
    pg_hook.run(f"DELETE FROM {self.table}")
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
    cur.execute(s)
psycopg2.ProgrammingError: relation "dependencies" does not exist

[2019-11-17 09:12:24,832] {taskinstance.py:1074} INFO - Marking task as UP_FOR_RETRY
[2019-11-17 09:12:24,839] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table Traceback (most recent call last):
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/bin/airflow", line 32, in <module>
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     args.func(args)
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     return f(*args, **kwargs)
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     _run(args, dag, ti)
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     pool=args.pool,
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     return func(*args, **kwargs)
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     result = task_copy.execute(context=context)
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 38, in execute
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     pg_hook.run(f"DELETE FROM {self.table}")
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table     cur.execute(s)
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table psycopg2.ProgrammingError: relation "dependencies" does not exist
[2019-11-17 09:12:24,840] {base_task_runner.py:115} INFO - Job 206: Subtask Load_dependencies_dim_table 
[2019-11-17 09:12:27,354] {logging_mixin.py:95} INFO - [[34m2019-11-17 09:12:27,353[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
